### PR TITLE
Simplification: Further Pruning

### PR DESCRIPTION
**Pull request by devlyn37**
## Description

This PR removes unused or redundant pieces from the SDK.

## Test plan

TS Compiling & Existing Tests

### CODE REVIEW

File: packages/client/wallets/aa/src/blockchain/BlockchainNetwoks.test.ts
Comment (devlyn37) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetwoks.test.ts:1]: these tests all might as well read:

```TS
// In file x.ts
const foo = 5;

// in file x.test.ts
expect(foo).to.be(5);
```

Comment (devlyn37) [reply packages/client/wallets/aa/src/blockchain/BlockchainNetwoks.test.ts:1]: You'll notice we weren't maintaining these tests either, see how most chains are missing

File: packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts [hunk 1]
```diff
@@ -42,39 +41,6 @@ import { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
 
 import { logInputOutput } from "../utils/log";
 
-export function getFireblocksAssetId(chain: EVMBlockchainIncludingTestnet) {
```
Comment (devlyn37) [inline packages/client/wallets/aa/src/blockchain/BlockchainNetworks.ts:45]: Not used anymore

File: packages/client/wallets/aa/src/blockchain/plugins.ts
Comment (devlyn37) [inline packages/client/wallets/aa/src/blockchain/plugins.ts:1]: Non existent functionality atm

**Review by maxisch — APPROVED** [review]

**Pull request merged**